client/components/payment-method-fees-pill/index.js

```diff
@@ -2,7 +2,7 @@ import { __, sprintf } from '@wordpress/i18n';
 import React from 'react';
 import Pill from '../pill';
 
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
In my code editor, I get this warning for this line.

![type](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/e3f5e40f-1bcc-4630-9816-47aa26433584)

This goes away if I add the following to my `eslintrc.js`


```extends: [ 'plugin:@woocommerce/eslint-plugin/recommended', 'plugin:@typescript-eslint/recommended' ]```

Reply:
This was not happening for me, but I added your suggestion and updated the rules file to ignore some other stuff that started to popup with this change 👍 . Thanks!